packages/deployment/app/data_storage.template.yml

```diff
@@ -0,0 +1,229 @@
+AWSTemplateFormatVersion: '2010-09-09'
```

Comment:
I *love* this change. 

Now if we can get this into it's own deployment stack entirely and make the outputs something that's config driven, we can make it *far* easier to resolve CF deployment issues.

💯